server.elm

```diff
@@ -2033,72 +2270,310 @@ editionscript = """
+          
+          Check siblings: When the siblings in status 2 is clicked in selector, the sibling will become 'clicked element' but the selector won't switch status 1. 
+          The siblings will be in the middle of second level of status 2. This is because we want user switching siblings in second level easily.
+
+          When other elements in selector are clicked, change 'clicked element' to it. And it also follow rules above.
+        */
+        domSelectorDiv.classList.add("dom-selector-style");
+        domSelectorDiv.append(
+          el("div", {"class": "mainElem"}, []),
+          el("div", {"class": "childrenElem"}, [])
+        );
+
+        let displayMainElem = function(elem) {
+          let mainElemDiv = document.querySelector(".dom-selector > .mainElem");
+          mainElemDiv.append(
+            el("div", {"class":"mainElemName", "type":"text", value: elem.tagName.toLowerCase()}, "<" + elem.tagName.toLowerCase() + ">", {
+              onmouseenter: (c => () => { c.setAttribute("ghost-hovered", "true") })(elem),
+              onmouseleave: (c => () => { c.removeAttribute("ghost-hovered") })(elem)
+            }),
+            el("div", {"class": "mainElemInfo"}, textPreview(elem, 50)) // display its text content
+          )
+        }
+
+        let displayChildrenElem = function(elem) {
+          let childrenElemDiv = document.querySelector(".dom-selector > .childrenElem");
+          childrenElemDiv.append(
+            el("div", {"class": "childrenSelector"},
+              [
+                el("div", {"class": "childrenSelectorName"}, "<" + elem.tagName.toLowerCase() + ">", {}),
+                el("div", {"class": "childrenSelectorInfo"}, textPreview(elem, 20))
+              ], 
+              {
+                onmouseenter: (c => () => { c.setAttribute("ghost-hovered", "true") })(elem),
+                onmouseleave: (c => () => { c.removeAttribute("ghost-hovered") })(elem)
+              }
+            )
+          );
+        }
+
+        let displayElemAttr = function(targetDiv, elem) {
+          // show attribute of element on the dom selector
+          for (let i = 0; elem && elem.attributes && i < elem.attributes.length; i++) {
+            let name = elem.attributes[i].name;
+            let value = elem.attributes[i].value;
+            if (name === "ghost-clicked" || name === "ghost-hovered") continue;
+            targetDiv.append(
+              el("div", { "class": "elementAttr" },
+                [
+                  el("span", { title: "This element has attribute name '" + name + "'" }, name + ": "),
+                  el("span", { title: "This element has attribute value '" + value + "'" }, value)
+                ]
+              )
+            );
+          }
+        }
+
+        if (clickedElem.id !== "context-menu" || clickedElem.id !== "modify-menu" || clickedElem.id !== "editbox") {
+          if (selectorStatus === 1) {
```

Comment:
Can you please
1) Put and initialize `selectorStatus` into `editor_model` (and access it from here using `model.selectorStatus`
2) Rename the integer `selectorStatus` to a more meaningful boolean name like `displaySelectedChildren` (true is like selectorStatus = 1). Alternatively, if you prefer, you could also store a string like "as child" or "as parent" in a variable such as `selectionDisplayModeClicked` 
3) Make sure this parameter is restored if the page reloads via the advanced menu. Have a look at other fields of `editor_model` for inspiration